src/const_eval.md

```diff
@@ -76,6 +77,10 @@ Notable features that const contexts have, but const fn haven't are:
 * union field access
 * [`transmute`] invocations.
 
+Conversely, the following are possible in a const function, but not in a const context:
+
+* Use of generic parameters.
```

Comment:
that's not possible in `const fn` yet either.

Reply:
Hm, maybe I'm a bit confused. The following seems to work for me:

```rust
const fn cfunc<T>(x: &T) -> usize {
    std::mem::size_of::<T>()
}
```

Reply:
ah, well, yes. I was thinking of trait bounds on generic parameters.

Reply:
I think the restriction on trait bounds is already mentioned above.  Let me know if you think this should be worded any differently.